cves/CVE-2013-1665.yml

```diff
@@ -49,7 +49,17 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The vulnerability exists in the way XML parsers and libraries used by Django
+  handle XML deserialization.
+  The implementation, which takes an XML and constructs into an object from the
+  structured data, does not handle parsing incorrectly. The DOCTYPE element
+  of XML can be injected with dangerously customized entities that are loaded
+  outside of the application.
+  Attackers can take advantage of this vulnerability by injecting network requests
+  via entity declaration in the DOCTYPE element of XML, forwarding and rebouding
+  with the server, and finally gaining read access to files. The server may also
+  take long time to process those requests, resulting in a DoS attack.
```

Comment:
This is a well explained and detailed description of the vulnerability. There are a couple of grammar errors.

Reply:
Thank you. Will fix the grammar. Might be even more helpful if you can pin point the grammar mistakes.